content/docs/100-get-started/200-build-an-app.mdx

```diff
@@ -17,7 +17,7 @@ Once you have an account, sign in to the [Developer Portal](https://developers.r
 
 From the [Apps](https://developers.raydiant.com/apps) section, we’ll start by creating the Simple Weather app and adding an icon and thumbnail.
 
-1. Click the “Create app” button [TODO: add icon].
+1. Click the “Create app” button <RaydiantIcon name="add" />.
```

Comment:
I think the icons are just slightly too big and dominating. What do you think about changing the size from 1.5rem to 1.25rem?

Before:
<img width="338" alt="image" src="https://user-images.githubusercontent.com/1163252/175349294-a6f9af59-5e85-4055-bc23-471f48cb1760.png">
<img width="402" alt="image" src="https://user-images.githubusercontent.com/1163252/175349237-51c2f4eb-542a-470b-8a71-69717e41f2b4.png">

After:
<img width="319" alt="image" src="https://user-images.githubusercontent.com/1163252/175349417-6330dbfe-10ee-431e-aac6-5735fa6cee14.png">
<img width="386" alt="image" src="https://user-images.githubusercontent.com/1163252/175349502-5916aaae-6624-4ae9-811f-c4d036d871a6.png">

I'm also debating surrounding the icons in brackets, what do you think?

<img width="398" alt="image" src="https://user-images.githubusercontent.com/1163252/175349687-d5a877b1-f154-4058-bb85-042b3fb80232.png">

Reply:
Yeah, the icons are definitely too big - good call!

I'm more of a minimalist, so i'd say without the brackets, but honestly don't have a strong preference. Lemme know if you think the brackets are better and I'll update.